pkg/java/prepare/prepare.go

```diff
@@ -44,30 +45,42 @@ func Prepare(dockerSpec config.DockerSpec, auroraVersions *runtime.AuroraVersion
 		return "", errors.Wrap(err, "Failed to read application metadata")
 	}
 
-	// Runtime scripts
-	if err := copyDefaultRuntimeScripts(dockerBuildPath); err != nil {
-		return "", errors.Wrap(err, "Failed to add static content to Docker context")
-	}
+	fileWriter := util.NewFileWriter(dockerBuildPath)
 
-	// Prepare application
-	if err := prepareEffectiveScripts(applicationFolder, meta); err != nil {
-		return "", errors.Wrap(err, "Failed to prepare application")
-	}
+	if architecture, exists := baseImage.ImageInfo.Labels["www.skatteetaten.no-imageArchitecture"]; exists && architecture == "java" {
```

Comment:
Er dette en label som settes i base image, og avhengig av base image så vil enten new dockerfile med radish brukes eller ikke? I så fall, kan vi kalle sufix på label navn være radish eller noe?

Reply:
Navngivingen er litt shaky her..

Tanken var at image lables etter arkitektur, som f.eks. Java/Webleveransepakke/plain osv.  Jeg tenkte at vi på mellomlang sikt fjerner den koden som ikke forholder seg til label, slik at mye av koden for webleveransepakke og Java kan bli lik.

Radish var i utgangspunktet tenkt som en implementasjonsdetalj, siden det vil være baseimaget som er ansvarlig for hvordan applikasjonen startes. Men i dag så genereres radish.json, så det er kanskje ikke en så liten implementasjonsdetalj som jeg først antok.

Jeg syns imageArchitecture fortsatt skal være Java, for det sier noe om at imaget har mulighet til å kjøre Java. Men det kan være det kan være hensiktsmessig å ha en "starerArchitecture" i tillegg som sier radish.

Reply:
Beklager. En smule dyslektisk. Med nye øyner så virker dette fornuftig. Så fjerner vi NewLegacyDockerFile på sikt når alle er over på ny base image og vi fikke finner bruk for den. Så helt enig i at imageArchitecture skal være Java :)